src/TableRow.js

```diff
@@ -0,0 +1,32 @@
+import React, { useState } from "react";
+import moment from "moment";
+
+function TableRow({ result }) {
+  const [highlight, setHighlight] = useState(false);
```

Comment:
So you've got the right idea but it is useful to be aware of naming conventions for booleans in this section. If you have a variable that stores a boolean value like a number you could think of a name that starts with a `is` or `has` e.g. `const isEven = true;`. The variable name `isEven` sort of reads like a question

Reply:
So think about applying the above when you create the state variable on line 5 - what does this variable reflect about your UI ?